packages/trader/src/Stores/Modules/Trading/Helpers/contract-type.ts

```diff
@@ -639,7 +639,8 @@ export const ContractType = (() => {
     const getCancellation = (contract_type: string, cancellation_duration: string, symbol: string) => {
         const arr_cancellation_range: string[] =
             getPropertyValue(available_contract_types, [contract_type, 'config', 'cancellation_range']) || [];
-
+        const cached_multipliers_cancellation: string[] =
```

Comment:
cant use `arr_cancellation_range` ? Not sure if its necessary

Reply:
In `arr_cancellation_range` `contract_type` is used and in `cached_multipliers_cancellation` - specifically `multipliers`. This allow us to store data about multipliers (and deal cancellations) even if the chosen type is another (e.d. Rise/Fall)

Reply:
yes but 'cancellation_range' only exists in multipliers so i dont think u need to explicitely pass 'multipliers' in contract_type

Reply:
<img width="1060" alt="Screenshot 2023-10-09 at 7 07 55 PM" src="https://github.com/binary-com/deriv-app/assets/118344354/30eb38c4-0224-4f6d-abc0-21040ed11230">

Reply:
<img width="1412" alt="Screenshot 2023-10-09 at 7 09 10 PM" src="https://github.com/binary-com/deriv-app/assets/118344354/8386d7cb-a629-4f57-9fed-487b396a4b85">

@kate-deriv im getting the exact same thing

Reply:
please try and see if you get the array in `contract-type-info` with `cancellation_range_list`

Reply:
nevermind. Ignore my comments. I see why youve done this. well done!